### ISSUE TITLE
[FEATURE REQUEST] plants should die if placed on a unbreathable planet!

### ISSUE THREAD

**Issue by MSandro** [Advanced-Rocketry/AdvancedRocketry#1484]
Hi, it would be realy nice to have a config option which would enable that plants (saplings, seeds, ...) would be replace with something like a dead bush if those are placed on an planet where it is not posible to breath.

**Comment by MuteTiefling** [comment]
I too would like to see something similar. Perhaps not immediate death, but if there is no atmosphere or low pressure atmosphere, plants should not be placeable and should ideally pop off to prevent further growth. 

I'd really like to allow farming in space/un-inhabitable planets in the pack I'm working on, but only if the area is sealed. Currently having a bit of difficulty in replicating this.

**Comment by C1ffisme** [comment]
I think the best way to handle this would be similar to how water kills plants: just pop the plant and drop the correct drops. This way if you accidentally forget to change a CO2 filter, your plant-life isn't completely destroyed.

In addition to this:
- Grass and Mycelium should slowly revert to Dirt
- Leaves on trees will slowly wither away regardless of whether or not they are near logs.
- Water should either freeze or boil away depending on the planet's temperature.

**Comment by bloche14** [comment]
Would be nice if you could configure any block to act like this, so that you could disable blocks from other mods that shouldn't work in a vaccuum, like windmills for example. (Or even vanilla furnaces which shouldn't work without oxygen)

**Comment by MuteTiefling** [comment]
The code for torchBlock would actually already fit this quite well. 

With a little tweak, it would prevent placement of the crops in vacuum. It also already breaks the crops on pressure loss. But the way it's coded leads to those items being voided rather than dropped in world

The trouble, it seems, is that the code as written is looking for blocks/items with identical names. So if I but minecraft:chest in the list, it works because both the item and the block share the name.

If I use any crops, however, the system breaks down. For instance, beetroots as a block are minecraft:beetroots while beetroot seeds are minecraft:beetroot_seeds. So the placement routine doesn't detect it properly and allows me to plant. And the breaking routine correctly breaks the beetroots, but doesn't give an item back for them.

I'd therefore propose a very minor change to those routines and the structure of the config for torchBlocks: Make it a key/value list

```
S:torchBlocks <
        minecraft:beetroots, minecraft:beetroot_seeds
        minecraft:redstone_torch     
     >
```

If we type in a simple block, it works as currently, looking up the item name based on the block name. 

If we put in a second value, however, on the same line, it would use that as the item name instead of attempting to search for it on it's own.

**Comment by voidsong-dragonfly** [comment]
Fixed in 2.0.0, using materials
